src/SuperSocket.Channel.Kestrel/TransportPipeChannel.cs

```diff
@@ -0,0 +1,453 @@
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SuperSocket.ProtoBase;
+
+namespace SuperSocket.Channel.Kestrel
+{
+    public class TransportPipeChannel<TPackageInfo>
+        : ChannelBase<TPackageInfo>
```

Comment:
I think you can inherit PipeChannel instead. Most of code in this class would not be required because they have been implemented in PipeChannel already.